README.md

```diff
@@ -112,6 +153,7 @@ Naming conventions from [PEP8](https://pep8.org/#prescriptive-naming-conventions
 are used. To summarize, class names use CapWords; constants are ALL_UPPERCASE;
 most other variables, functions and methods are all_lowercase.
 
+
```

Comment:
My fault 😊 
I ended up seeing 2 blank lines before one of the level 2 heading (probably because I had copy/pasted stuff around) and then when I saw other headings with only 1 blank line, I decided to make them consistent!
I'll remove the extra blanks I inserted - I agree that there's no need for them.